src/shared/validators.js

```diff
@@ -17,15 +17,73 @@ const userPasswordUpperCheck = password => userPasswordUpperCaseValid.test(passw
 const userPasswordNumbersCheck = password => userPasswordNumbersValid.test(password);
 const userPasswordMinSymbolsCheck = password => userPasswordMinSymbolsValid.test(password);
 
-export const passwordPower = password => {
+const userNickname = document.getElementById('username');
+const userEmail = document.getElementById('email');
+const userPassword = document.getElementById('password');
+const userConfirmPassword = document.getElementById('confirmPassword');
+
+const lowerCaseCheck = password => {
+  const result = userPasswordLowerCaseValid.test(password);
+  return result;
+}
```

Comment:
it would be better to make it like one code line.

Reply:
fixed all